dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
+hide.messages.desc=The list of messages, separated by a comma, that are configured to be hidden from the console.log and message.log files. If the messages are configured to be hidden, then they are redirected to the trace.log file. Avoid trouble: Use the com.ibm.ws.logging.hide.messages property, except in cases where you are trying to change the configuration dynamically after startup.
 
 message.source=Message source
-message.source.desc=The list of sources to route to messages.log
+message.source.desc=The list of comma-separated sources that route to the messages.log file. This property only applies when messageFormat=json. Valid values are message, trace, accessLog, ffdc, and audit. By default, messageSource is set to the environment variable WLP_LOGGING_MESSAGE_SOURCE (if set), or message. Note: To use the audit source, enable the Liberty audit feature. Avoid trouble: Use the WLP_LOGGING_MESSAGE_SOURCE environment variable or com.ibm.ws.logging.message.source property, except in cases where you are trying to change the configuration dynamically after startup.
 
 message.format=Message format
-message.format.desc=The format to use for messages.log
+message.format.desc=The required format for the messages.log file. Valid values are basic or json format. By default, messageFormat is set to the environment variable WLP_LOGGING_MESSAGE_FORMAT (if set) or basic. Avoid trouble: Use the WLP_LOGGING_MESSAGE_FORMAT environment variable or com.ibm.ws.logging.message.format property, except in cases where you are trying to change the configuration dynamically after startup.
 
 console.source=Console source
-console.source.desc=The list of sources to route to console.log / console
+console.source.desc=The list of comma separated sources that route to the console/console.log. This property only applies when consoleFormat=json. Valid values are message, trace, accessLog, ffdc, and audit. By default, consoleSource is set to the environment variable WLP_LOGGING_CONSOLE_SOURCE (if set), or message. Note: To use the audit source, enable the Liberty audit feature. Avoid trouble: Use the WLP_LOGGING_CONSOLE_SOURCE environment variable or com.ibm.ws.logging.console.source property, except in cases where you are trying to change the configuration dynamically after startup.
 
 console.format=Console format
-console.format.desc=The format to use for console.log / console
+console.format.desc=The required format for the console. Valid values are basic or json format. By default, consoleFormat is set to the environment variable WLP_LOGGING_CONSOLE_FORMAT (if set) or basic. Avoid trouble: Use the WLP_LOGGING_CONSOLE_FORMAT environment variable or com.ibm.ws.logging.console.format property, except in cases where you are trying to change the configuration dynamically after startup.
```

Comment:
Change
"Valid values are basic or json format. By default, consoleFormat is set to the environment variable WLP_LOGGING_CONSOLE_FORMAT (if set) or basic. Avoid trouble: Use the WLP_LOGGING_CONSOLE_FORMAT environment variable or com.ibm.ws.logging.console.format property, except in cases where you are trying to change the configuration dynamically after startup."
to
"Valid values are basic or JSON format. By default, consoleFormat is set to the environment variable WLP_LOGGING_CONSOLE_FORMAT (if set), or basic. Avoid trouble: Use the WLP_LOGGING_CONSOLE_FORMAT environment variable or the com.ibm.ws.logging.console.format property, except in cases where you are trying to change the configuration dynamically after startup."